mappings/net/minecraft/client/gui/screen/option/OptionsScreen.mapping

```diff
@@ -1 +1,7 @@
 CLASS net/minecraft/class_635 net/minecraft/client/gui/screen/option/OptionsScreen
+	FIELD field_2761 title Ljava/lang/String;
+	FIELD field_2762 parent Lnet/minecraft/class_32;
+	FIELD field_2763 settings Lnet/minecraft/class_322;
```

Comment:
`options` would probably make more sense, especially considering it's named that in `VideoOptionsScreen`